scripts/populate_institutions.py

```diff
@@ -983,11 +983,11 @@ def main(default_args=False):
                 'description': 'A research data service provided by the BTNRH Research Technology Core. Please do not use this service to store or transfer personally identifiable information or personal health information. For assistance please contact <a href="mailto:Christine.Hammans@boystown.org">Christine.Hammans@boystown.org</a>.',
                 'banner_name': 'bt-banner.png',
                 'logo_name': 'bt-shield.png',
-                'login_url': None,
-                'logout_url': None,
+                'login_url': SHIBBOLETH_SP_LOGIN.format(encode_uri_component('https://sts.windows.net/e2ab7419-36ab-4a95-a19f-ee90b6a9b8ac/')),
```

Comment:
For Boys Town, its Test and Prod IdP shares the same entity ID but the metadata file for each is different.